tests/parsers/plist_plugins/ios_siminfo.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+from plaso.parsers.plist_plugins import ios_siminfo
+from tests.parsers.plist_plugins import test_lib
+
+
+class IOSSIMInfoPluginTest(test_lib.PlistPluginTestCase):
+    """Test untuk plugin iOS SIM Info."""
+
+    def testProcess(self):
+        """Menguji proses parsing plist menggunakan plugin iOS SIM Info."""
+        # Nama file plist yang akan diuji
+        plist_name = 'com.apple.commcenter.data.plist'
+
+        # Membuat instance dari plugin
+        plugin = ios_siminfo.IOSSIMInfoPlugin()
+
+        # Memproses file plist dengan plugin
+        storage_writer = self._ParsePlistFileWithPlugin(plugin, [plist_name], plist_name)
+
+        # Mengambil jumlah event_data yang diproses
+        number_of_event_data = storage_writer.GetNumberOfAttributeContainers('event_data')
+
+        # Memastikan jumlah event_data yang diproses sesuai
+        self.assertEqual(number_of_event_data, 1)
+
+        # Mengambil daftar event_data
+        events = list(storage_writer.GetAttributeContainers('event_data'))
+
+        # Memastikan bahwa data di event pertama sesuai dengan yang diharapkan
+        event = events[0]
```

Comment:
Please use expected_event_values to match rest of codebase e.g. see ios_carplay.py for an example